iina/KeyBindingTranslator.swift

```diff
@@ -114,12 +119,16 @@ class KeyBindingTranslator {
   static func string(fromCriterions criterions: [Criterion]) -> String {
     var mapped = criterions.filter { !$0.isPlaceholder }.map { $0.mpvCommandValue }
 
-    let firstName = (criterions[0] as! TextCriterion).name
+    let firstCriterion = criterions[0] as! TextCriterion
```

Comment:
I think `first` is designed to be used when we don't know whether the array is empty. Here it must have a first element; and if not, although I can't image such scenario, crash will be a better way to track what happened.